bot/seasons/halloween/spookyrating.py

```diff
@@ -0,0 +1,77 @@
+import bisect
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+from bot.constants import Colours
+
+log = logging.getLogger(__name__)
+
+with Path('bot', 'resources', 'halloween', 'spooky_rating.json').open() as file:
+    SPOOKY_DATA = json.load(file)
+    SPOOKY_DATA = sorted((int(key), value) for key, value in SPOOKY_DATA.items())
+
+
+class SpookyRating:
+    """
+    A cog for calculating one's spooky rating
+    """
+
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    @commands.cooldown(rate=1, per=5, type=commands.BucketType.user)
+    async def spookyrating(self, ctx, who: discord.Member = None):
+        """
+        Calculates the spooky rating of someone.
+
+        This command accepts an optional user as an argument, defaulting to the command executor.
+        Users are converted from:
+          - User ID
+          - Mention
+          - name#discrim
+          - name
+          - nickname
```

Comment:
I don't think we need to document this. Otherwise, to stay consistent, we would need this on every function that takes a member as input.

Reply:
I mean I saw it on one already - I just copied and pasted it. I can remove it if necessary

Reply:
Sorry, I missed that. But yeah, would be great if you could remove this.